src/ControlSystem/PiecewisePolynomial.hpp

```diff
@@ -58,9 +58,8 @@ class PiecewisePolynomial {
     DerivInfo(double t, value_type deriv) noexcept;
   };
 
-  /// Returns a DerivInfo corresponding to the first element in the range of
-  /// DerivInfos with an update time that is greater than `t`, or the last
-  /// DerivInfo if no such element is found.
+  /// Returns a DerivInfo corresponding to the last element in the range of
```

Comment:
See the last comment on this

Reply:
What you define first and last depends on where you choose to start the range, I start it at `t`, others start it at `t_0`, this is why we asked for closest.